src/Microsoft.Diagnostics.DotNetMonitor.Core/Startup.cs

```diff
@@ -38,7 +39,8 @@ public void ConfigureServices(IServiceCollection services)
                 System.Reflection.PropertyInfo prop = options.GetType().GetProperty("EnableEndpointRouting");
                 prop?.SetValue(options, false);
 
-            }).SetCompatibilityVersion(CompatibilityVersion.Latest);
+            }).SetCompatibilityVersion(CompatibilityVersion.Latest)
+                .AddApplicationPart(typeof(DiagController).Assembly);
```

Comment:
Without this registration, the controllers in the REST server assembly are not automatically discovered.

Reply:
Any idea why this is the case now, given that previously they were already discovered and were in a separate assembly?